### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by AustineA** [ojengwa/paystackr#1]
But can you create how to use documentation?
I mean what's worth doing is worth doing well... Without documentation how do you expect people to use this? Unless your intention is that, then i completely understand.

**Comment by Owohlet** [comment]
I agree... with @AustineA, I mean I got excited Paystack has a gem only to see it doesn't actually have a proper doc.
